jep/4/README.adoc

```diff
@@ -147,21 +163,21 @@ Create Google Groups at
 link:https://groups.google.com/forum/#!creategroup[],
 following the procedure:
 
-* Each SIG should have one discussion groups, and a number of groups for mirroring relevant github notifications;
+* Each SIG should have at least one discussion group.
+This group should be added to the SIG metadata
+* SIGs may also have a number of groups for mirroring relevant github notifications;
 * Create groups using the name conventions below;
 * Groups should be created as e-mail lists with at least three owners
   (including tyler at monkeypox.org and verninol at gmail.com to ensure SIG
   continuity);
 * To add the owners, visit the Group Settings (drop-down menu on the right
   side), select Direct Add Members on the left side and add Tyler and Olivier
   via email address (with a suitable welcome message); in Members/All Members
-  select Tyler and Olivier and assign t hem to an "owner role" for long term
+  select Tyler and Olivier and assign them to an "owner role" for long term
   maintenance.
 * Set "View topics", "Post", "Join the Group" permissions to be "Public"
 
-Name convention:
-
-* jenkins-foo (the discussion group)
+Naming convention: `jenkins-${sigId}` (the discussion group)
```

Comment:
The existing groups are name `jenkinsci-<etc>`.   Is the difference here intentional?

Reply:
I followed the original naming format from @rtyler . Generally jenkins vs. jenkinsci is a long-long story. I have no preference., both fine with me

Reply:
So far we've used `jenkins-${sigId}-sig`.  Which seems fine to me.